lib/src/main/java/com/ibm/cloud/appid/android/internal/tokenmanager/TokenManager.java

```diff
@@ -75,6 +77,12 @@
 	private final static String ERROR_DESCRIPTION = "error_description";
 	private final static String ERROR = "error";
 	private final static String INVALID_GRANT = "invalid_grant";
+
+	private static final String OAUTH_ACTIVITY_LOGGING_PATH = "/activity_logging";
+	private static final String ID_KEY = "id_token";
+	private static final String ACTIVITY_KEY = "eventName";
```

Comment:
can we name it
EVENT_NAME  and
ID_TOKEN ?

Reply:
we can't, that's how the json is defined in the spec

Reply:
changed to EVENT_NAME_KEY, ID_TOKEN_KEY